robot/Cumulus/resources/NPSP.py

```diff
@@ -739,6 +747,8 @@ def click_button_with_value (self,title):
         """clicks on the button on the payments page"""      
         locator=npsp_lex_locators['button'].format(title)
         self.selenium.get_webelement(locator).click()
+        if title == "Create Payments":
+            self.selenium.wait_until_page_does_not_contain_element(locator, error="Clicking Button "+ title +" was not successful")
```

Comment:
You can just do `Wait Until Page Does Not Contain Element | title=Create Payments` after the line in the test suite that calls this keyword. I think it would be better to keep this generic keyword clean.

Reply:
removed this and I am using Wait Until Page Does Not Contain Element with xpath